mantidproject/mantid
# Move vanadium processing onto focus tab of Engineering UI

**Description of work.**

The generation of the files required for Vanadium normalization is now done on the Focus tab of the user interface. This means the Vanadium data can be updated without having to rerun the Ceria calibration.

This PR includes a few other related changes:
- the setting "Force Vanadium Recalculation" has been removed and the Vanadium run number input has been moved from the Calibration tab to the Focus tab
- The Vanadium run number is now part of the processed vanadium workspace names in the ADS
- The Vanadium run number is also no longer written to the prm generated on the Calibration tab.

**To test:**

Open the engineering diffraction user interface (Interfaces, Diffraction menu)
Go to the Calibration tab
Run a calibration by selecting the "Create New Calibration" radio button and use Calibration Sample 193749
Go to the Focus tab
The region of interest should say "North and South banks"
Enter Vanadium 236516, Sample 299080
Hit Focus button

Check this all completes without errors

Fixes #31904. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
